server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -70,6 +70,10 @@ class PullRequestMiner:
 
     CACHE_TTL = 5 * 60
     log = logging.getLogger("%s.PullRequestMiner" % metadata.__package__)
+    mappers = {
```

Comment:
The life has taught me that explicit types are better:
```suggestion
    AuxiliaryMappers = namedtuple("AuxiliaryMappers", ["releases_to_prs", "prs_to_releases"])
    mappers = AuxiliaryMappers(
        releases_to_prs=ReleaseToPullRequestMapper.map_releases_to_prs,
        prs_to_releases=PullRequestToReleaseMapper.map_prs_to_releases,
    )
```
I suffered from my previous dict-like scheme in `entries.py` while refactoring code in PyCharm.

Reply:
ah good idea in using `namedtuple` 👍